docs/qgis_plugin.md

```diff
@@ -1,14 +1,20 @@
-# QGIS Plugin
+# QGIS2Wegue QGIS Plugin
 
-Adding many layers to a configuration file can be laborious, this is where `QGIS2Wegue` ([GitHub Repository](https://github.com/meggsimum/qgis2wegue)) comes in handy. It converts all layers of a QGIS project into a valid Wegue configuration. This works for `WMS`, `XYZ`, `KML`, `GeoJSON`, `WFS`.
+Configuring your Wegue application by hand can be laborious. This is where QGIS plugin `QGIS2Wegue` ([GitHub Repository](https://github.com/meggsimum/qgis2wegue)) comes in handy. It derives a valid Wegue application configuration from your QGIS project:
+
+  - All layers of a QGIS project are converted into a valid Wegue layer configuration. This works for `WMS`, `XYZ`, `KML`, `GeoJSON`, `WFS`
+  - The map extent is taken over to the configuration
+  - Textual elements, like title and footer can be set in a GUI
+  - The modules, which should be included in your Wegue application, be easily be added / removed via GUI
 
```

Comment:
Editorial: 
Modules to be included in your Wegue application can be easily added/removed via the GUI

Reply:
yes, that is a better phrasing. 
@chrismayer could you replace this sentence with Just's suggestion?
-->  `Modules to be included in your Wegue application can be easily added/removed via the GUI`